File: SiteScraper/pcbmodel.py
class PCB:
    def __init__(self):
        self.id = None
        self.manufacturer = None
        self.manufacturerLogo = None
        self.pcbName = None
        self.pcbNotes = None
        self.pcbImages = []
        self.lifeSpan = None
        self.pcbClass = None
        self.mapper = None
        self.prgRom = None
        self.prgRam = None
        self.chrRom = None
        self.chrRam = None
        self.batteryPresent = None
        self.mirroring = None
        self.cic = None
        self.otherChips = None

    def to_dict(self):
        return {
            "id": self.id,
            "manufacturer": self.manufacturer,
            "manufacturerLogo": self.manufacturerLogo,
            "pcbName": self.pcbName,
            "pcbNotes": self.pcbNotes,
            "pcbImages": self.pcbImages,
            "lifeSpan": self.lifeSpan,
            "pcbClass": self.pcbClass,
            "mapper": self.mapper,
            "prgRom": self.prgRom,
            "prgRam": self.prgRam,
            "chrRom": self.chrRom,
            "chrRam": self.chrRam,
            "batteryPresent": self.batteryPresent,
            "mirroring": self.mirroring,
            "cic": self.cic,
            "otherChips": self.otherChips,
        }

File: CartDB.Parser/Models/PcbModel.cs
using System.Collections.Generic;

namespace CartDB.Parser.Models
{
    public class PcbModel
    {
        public string Manufacturer { get; set; }
        public string ManufacturerLogo { get; set; }
        public string PcbName { get; set; }
        public string PcbNotes { get; set; }
        public List<string> PcbImages { get; set; }
        public string LifeSpan { get; set; }
        public string PcbClass { get; set; }
        public string Mapper { get; set; }
        public string PRGRom { get; set; }
        public string PRGRam { get; set; }
        public string CHRRom { get; set; }
        public string CHRRam { get; set; }
        public string BatteryPresent { get; set; }
        public string Mirroring { get; set; }
        public string CIC { get; set; }
        public string OtherChips { get; set; }
    }

    public class PcbListModel
    {
        public List<PcbModel> Pcbs { get; set; }
    }
}
